extension/data/frame/frame.html

```diff
@@ -2,28 +2,26 @@
 <html>
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <link rel="stylesheet" href="widget.css" type="text/css" />
+  <link rel="stylesheet" href="frame.css" type="text/css" />
```

Comment:
As you can see, I didn't copied `widget.css` to `frame.css`.
Somebody should write a new CSS, because the old looks ugly on toolbar.